python/cugraph/cugraph/tests/data_store/test_property_graph.py

```diff
@@ -2576,9 +2576,12 @@ def bench_extract_subgraph_for_rmat(gpubenchmark, rmat_PropertyGraph):
     scn = PropertyGraph.src_col_name
     dcn = PropertyGraph.dst_col_name
 
+    # Build a query string to extract a graph with only specific edges based on
+    # the integer vertex IDs. Other edge and/or vertex properties can be
+    # included in the query as well.
     verts = []
     for i in range(0, 10000, 10):
```

Comment:
```suggestion
```

Reply:
Thanks, good suggestions, I just pushed a commit to do this and the others below.